src/command-parser/parserConfig.ts

```diff
@@ -161,25 +165,29 @@ export const parserConfig = {
     'key-gen': keyGenArgs,
 
     'show': { // hw-specific subpath
-      '--payment-path': {
-        type: (path: string) => parseBIP32Path(path),
-        dest: 'paymentPath',
-        help: 'Payment derivation path. (specify only path or script hash)',
-      },
-      '--payment-script-hash': {
-        type: (hashHex: string) => parseScriptHashHex(hashHex),
-        dest: 'paymentScriptHash',
-        help: 'Payment derivation script hash in hex format.',
-      },
-      '--staking-path': {
-        type: (path: string) => parseBIP32Path(path),
-        dest: 'stakingPath',
-        help: 'Stake derivation path. (specify only path or script hash)',
+      '_mutually-exclusive-group-payment-required': {
```

Comment:
I'd consider naming them
```
_mutually-exclusive-group-optional-*
_mutually-exclusive-group-required-*
```

Reply:
Changed in https://github.com/vacuumlabs/cardano-hw-cli/pull/109/commits/955aadbe22fe208731b7671a099156c7cf91eb1e